migrations/2025-04-10-234244_add_liked_combined/up.sql

```diff
@@ -0,0 +1,45 @@
+-- Creates combined tables for
+-- person_liked: (comment, post)
+--
+-- This one is special, because you use the liked date, not the ordinary published
+CREATE TABLE person_liked_combined (
+    id serial PRIMARY KEY,
+    liked timestamptz NOT NULL,
```

Comment:
This table will get very large if it has to store all vote timestamps of all users. Hopefully it is only stored for local users? And it would be better to get the timestamp via join to post_action / comment_action table. Also the  column name is unclear.

Reply:
Still it should be possible to sort by the timestamp from post_actions table as it gets joined anyway.